package.json

```diff
@@ -6,6 +6,7 @@
     "author": "Francesco Agosti <francesco@0x.org>",
     "license": "Apache-2.0",
     "scripts": {
+        "install": "sed -ie 's!import { Provider as Web3Provider } from \"web3/providers\";!import { Web3EthereumProvider as Web3Provider } from \"web3-providers\";!' node_modules/ganache-core/typings/index.d.ts # see https://github.com/trufflesuite/ganache-core/issues/465#issuecomment-610005598",
```

Comment:
this can be deleted now, right? since we're not using migrations any more (now that we ripped out the contract deploy from the tests), so we don't really need this... right?

Reply:
No, this can't be deleted, because `ganache-core` is pulled in via `@0x/subproviders`.